src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -212,25 +213,10 @@ void SolverInterfaceImpl::configure(
       config.getCouplingSchemeConfiguration();
   _couplingScheme = cplSchemeConfig->getCouplingScheme(_accessorName);
 
-  // Add meshIDs and data IDs
-  for (const MeshContext *meshContext : _accessor->usedMeshContexts()) {
-    const mesh::PtrMesh &mesh   = meshContext->mesh;
-    const auto           meshID = mesh->getID();
-    _meshIDs[mesh->getName()]   = meshID;
-    PRECICE_ASSERT(_dataIDs.find(meshID) == _dataIDs.end());
-    _dataIDs[meshID] = std::map<std::string, int>();
-    PRECICE_ASSERT(_dataIDs.find(meshID) != _dataIDs.end());
-    for (const mesh::PtrData &data : mesh->data()) {
-      PRECICE_ASSERT(_dataIDs[meshID].find(data->getName()) == _dataIDs[meshID].end());
-      _dataIDs[meshID][data->getName()] = data->getID();
-    }
-    std::string                meshName   = mesh->getName();
-    mesh::PtrMeshConfiguration meshConfig = config.getMeshConfiguration();
-  }
   // Register all MeshIds to the lock, but unlock them straight away as
   // writing is allowed after configuration.
-  for (const auto &meshID : _meshIDs) {
-    _meshLock.add(meshID.second, false);
+  for (const MeshContext *meshContext : _accessor->usedMeshContexts()) {
```

Comment:
Why not using `const auto &meshContext` as it is done for other loops?

Reply:
Ok. I agree on making things explicit and I will disable `modernize-use-auto` for the tidy checks.